content/getting-started/finding-kata.md

```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
 
 # Finding Kata
 
-<!---
-TODO training suggestions
-TODO kata search page
-TODO kata "numbers": rank, satisfaction, possibly others (which?) as criteria for selecting a kata to solve
--->
+Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.
+
+## Codewars trainer
```

Comment:
```suggestion
## Codewars training suggestions
```

Reply:
It's called "Personal Trainer" internally. Like those in gyms.